src/main/java/com/to/ordermanagementservice/BasketTest.java

```diff
@@ -0,0 +1,36 @@
+package com.to.ordermanagementservice;
```

Comment:
GH Issue ID in PR name does not correspond to the task.

Reply:
what do you mean ? can you explain me .

Reply:
Sure.

PR Name: **gh-5 Create different types of Products (Part 1)**

GH Issue: **Create different types of Products (Part 1) #8**
URL: https://github.com/Andrii77711/order-management-service/issues/8

Reply:
5 vs 8

Reply:
do i need create new branch ?

Reply:
let's just update PR name this time